src/components/molecules/Settings/NavigationItem/index.tsx

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import { styled, fonts } from "@reearth/theme";
 import { Link } from "@reach/router";
+import { colors } from "@reearth/theme";
```

Comment:
Again sorry looks like we won't be importing the colors directly when we have multiple themes. Same as above comment, use colors directly from theme